WarpplateManager.cs

```diff
@@ -74,9 +74,9 @@ public string InAreaWarpplateName(int x, int y)
         /// </summary>
         /// <param name="worldid">World name to get Warpplates from</param>
         /// <returns>List of Warpplates with only their names</returns>
-        public List<Warpplate> ListAllWarpplates(string worldid)
+        public List<String> ListAllWarpplates()
         {
-            var WarpplatesTemp = new List<Warpplate>();
+            /*var WarpplatesTemp = new List<Warpplate>();
```

Comment:
You should probably delete instead of commenting out code that has been replaced.

Reply:
Resolved in c68d4b0fd1334e48c192e65f101f8c90565f911c